src/driver/amdxdna/ve2_host_queue.h

```diff
@@ -105,6 +105,7 @@ struct ve2_hsa_queue {
 	struct hsa_queue		*hsa_queue_p;
 	struct ve2_mem			hsa_queue_mem;
 	struct ve2_hq_complete		hq_complete;
+	struct mutex			hq_lock;
```

Comment:
Checkpatch will fail about this line. All of the lock needs a comment to explain what the lock is trying to protect.